tools/block-generator/Makefile

```diff
@@ -1,8 +1,9 @@
-SCENARIO = scenarios/config.app.create.yml # test_config.yml
+SCENARIO = scenarios/config.allmixed.small.yml
```

Comment:
This Makefile is to ease local testing. It includes various hacks (such as `make post-git-push`) that reduce the friction of local development and make my IDE happier. I'm happy modify or even remove the `Makefile` if I receive any such feedback.